process-automation/latest/model/scripts.md

```diff
@@ -40,7 +40,9 @@ To create a script:
 
 The Modeling Application contains two tabs for creating and managing scripts.
 
-The **Script Editor** is the GUI for modeling scripts by typing in the declared language. The **Metadata** contains the properties related to the script.
+The **Script Editor** is the GUI for modeling scripts by typing in the declared language. The editor has autocomplete functionality for APIs and script variables. The **Metadata** contains the properties related to the script.
+
+![auto complete]({% link process-automation/images/auto-complete.png %})
```

Comment:
Maintenance consideration: we should standardize the term: autocomplete / auto complete / auto-complete even though they're not all visible (future consideration).